src/tasks/tasks.proj

```diff
@@ -24,7 +24,7 @@
     </ItemGroup>
 
     <MSBuild Projects="@(TaskProject)"
-             Properties="Configuration=Debug;Platform=AnyCPU"
+             Properties="Configuration=$(TasksConfiguration);Platform=AnyCPU"
```

Comment:
The ProjectToBuild.AdditionalProperties from Subset.props won't be flowed the the individual task projects here, right? Should this also explicitly propagate `LibrariesConfiguration=$(LibrariesConfiguration)`?

Reply:
@ViktorHofer I think this is my only question here.
Everything else looks good.

Reply:
Oh wow, I completely dropped the ball here. Let me rebase on top of main, look at the binlog and then think about your question. Thanks for the ping.